### PR TITLE
DOC-4304: Added the default isolation level for database transactions in state persistence doc

### PR DESCRIPTION
**Pull request by amiegrace12**

### CODE REVIEW

**Review by yarivus — DISMISSED** [review]
LGTM

**Review by yarivus — DISMISSED** [review]
LGTM

File: content/en/platform/corda/4.6/enterprise/cordapps/state-persistence.md [hunk 1]
```diff
@@ -771,6 +771,8 @@ A `withEntityManager` block has the following three outcomes, which are managed
 
 Changes are committed to the database when the transaction is committed.
 
+Configuration is set to `READ_COMMITED` by default, and it is not configurable. This is the best option for Corda operations at database Level.
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/4.6/enterprise/cordapps/state-persistence.md:774]:
```suggestion
The configuration is set to `READ_COMMITED` by default, and it is not configurable. This is the best option for Corda operations at the database level.
```

File: content/en/platform/corda/4.7/enterprise/cordapps/state-persistence.md [hunk 2]
```diff
@@ -771,6 +771,8 @@ A `withEntityManager` block has the following three outcomes, which are managed
 
 Changes are committed to the database when the transaction is committed.
 
+Configuration is set to `READ_COMMITED` by default, and it is not configurable. This is the best option for Corda operations at database Level.
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/4.7/enterprise/cordapps/state-persistence.md:774]:
```suggestion
The configuration is set to `READ_COMMITED` by default, and it is not configurable. This is the best option for Corda operations at the database level.
```

File: content/en/platform/corda/4.8/enterprise/cordapps/state-persistence.md [hunk 3]
```diff
@@ -745,6 +745,8 @@ A `withEntityManager` block has the following three outcomes, which are managed
 
 Changes are committed to the database when the transaction is committed.
 
+Configuration is set to `READ_COMMITED` by default, and it is not configurable. This is the best option for Corda operations at database Level.
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/4.8/enterprise/cordapps/state-persistence.md:748]:
```suggestion
The configuration is set to `READ_COMMITED` by default, and it is not configurable. This is the best option for Corda operations at the database level.
```

File: content/en/platform/corda/4.9/enterprise/cordapps/state-persistence.md [hunk 4]
```diff
@@ -745,6 +745,8 @@ A `withEntityManager` block has the following three outcomes, which are managed
 
 Changes are committed to the database when the transaction is committed.
 
+Configuration is set to `READ_COMMITED` by default, and it is not configurable. This is the best option for Corda operations at database Level.
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/4.9/enterprise/cordapps/state-persistence.md:748]:
```suggestion
The configuration is set to `READ_COMMITED` by default, and it is not configurable. This is the best option for Corda operations at the database level.
```

**Review by paulmoloneyr3 — CHANGES_REQUESTED** [review]

**Review by colin-hillis — CHANGES_REQUESTED** [review]

File: content/en/platform/corda/4.6/enterprise/cordapps/state-persistence.md [hunk 5]
```diff
@@ -771,6 +771,8 @@ A `withEntityManager` block has the following three outcomes, which are managed
 
 Changes are committed to the database when the transaction is committed.
 
+The configuration is set to `READ_COMMITED` by default, and it is not configurable. This is the best option for Corda operations at the database level.
```
[the remark this answers is not in the document]
Comment (amiegrace12) [reply content/en/platform/corda/4.6/enterprise/cordapps/state-persistence.md:774]: Just checked and it's a typo, thanks for spotting it!

**Review by yarivus — COMMENTED** [review]
LGTM
thanks @amiegrace12 & @colin-hillis

**Review by paulmoloneyr3 — APPROVED** [review]

**Review by colin-hillis — APPROVED** [review]
Thanks for fixing that Nadine.

**Pull request merged**